src/expressjs/controllers/products.js

```diff
@@ -1,33 +1,44 @@
-import * as products from '../config/products';
-import fs from 'fs';
-import path from 'path';
+import { models } from '../database/connection';
 
+const { Product } = models;
 export const getAllProducts = (req, res) => {
-    return res.json(products.default);
+    Product.findAll().then(products => {
+        console.log(`All products ${JSON.stringify(products)}`);
+        return res.json(products);
+    })
+    .catch(err => res.send(`Error: ${err}`));
 }
 
 export const getProduct = (req, res) => {
     const { id } = req.params;
-    const result = products.default.find(product => product.id === parseInt(id, 10));
-
-    return res.json(result);
+    Product.findByPk(id).then(product => {
+        console.log(`Product ${JSON.stringify(product)}`);
+        return res.json(product);
+    })
+    .catch(err => res.send(`Error: ${err}`));
 }
 
 export const getProductReviews = (req, res) => {
     const { id } = req.params;
 
-    const reviews = products.default.find(product => product.id === parseInt(id, 10)).reviews;
-    
-    return res.json(reviews);
+    const reviews = Product.findOne({
+        where: {
+            id: id
```

Comment:
please use ES6 shorthand object properties

Reply:
Fixed it now thanks